projects/opendr_ws/src/perception/scripts/object_detection_2d_gem.py

```diff
@@ -18,43 +18,43 @@
 import torch
 import message_filters
 import cv2
-import time
 import numpy as np
+import argparse
 from vision_msgs.msg import Detection2DArray
 from sensor_msgs.msg import Image as ROS_Image
 from opendr_bridge import ROSBridge
 from opendr.perception.object_detection_2d import GemLearner
-from opendr.perception.object_detection_2d import draw
+from opendr.perception.object_detection_2d import draw_bounding_boxes
 from opendr.engine.data import Image
 
 
-class GemNode:
-
-    def __init__(self,
-                 input_color_topic="/camera/color/image_raw",
-                 input_infra_topic="/camera/infra/image_raw",
-                 output_color_topic="/opendr/color_detection_annotated",
-                 output_infra_topic="/opendr/infra_detection_annotated",
-                 detection_annotations_topic="/opendr/detections",
-                 device="cuda",
-                 pts_color=None,
-                 pts_infra=None,
-                 ):
+class ObjectDetectionGemNode:
+    def __init__(
+        self,
+        input_color_image_topic="/camera/color/image_raw",
+        input_infra_image_topic="/camera/infra/image_raw",
+        output_color_image_topic="/opendr/color_detection_annotated",
```

Comment:
In an earlier commit "color" was changed to "rgb" as far as i can tell, but the changes were reverted.